scripts/installCommonDeps.sh

```diff
@@ -20,6 +20,17 @@ check_proxy(){
   fi
 }
 
+install_srt(){
+  local SRC_URL="https://github.com/Haivision/srt.git"
+  local SRC_DIR="srt"
+  mkdir -p ${LIB_DIR}
+  pushd ${LIB_DIR}
+  git clone ${SRC_URL}
```

Comment:
Better to use a certain version of SRT instead of the latest master

Reply:
Updated